frontend/src/views/FuelCodes/AddFuelCode/AddEditFuelCode.jsx

```diff
@@ -229,10 +231,12 @@ const AddEditFuelCodeBase = () => {
           const errMsg = `Error updating row: ${
             fieldLabels.length === 1 ? fieldLabels[0] : ''
           } ${message}`
-
+          updatedData.validationMsg = errMsg
```

Comment:
Tested and received 422, when processing the date fields. They are not in correct format. Then, needed to give a proper format. Something like this:
```const dateFields = ['applicationDate', 'approvalDate', 'effectiveDate', 'expirationDate'];

      // Format date fields to ISO 8601
      const formattedData = { ...params.node.data };
      dateFields.forEach((field) => {
        if (formattedData[field]) {
          const date = new Date(formattedData[field]);
          if (!isNaN(date.getTime())) {
            formattedData[field] = date.toISOString().split('T')[0]; // Keep only the YYYY-MM-DD part
          }
        }
      });
      let updatedData = Object.entries(formattedData)
```
After that the request passes the schema validation, it can start executing create_fuel_code. However, an error is raised: ValueError: Invalid transport mode: Truck

Reply:
Thanks for your review @areyeslo , 
- yes, from excel data needs to be copied in proper format otherwise it throws error.
- Truck is no longer among the valid transport mode in the database. hence value error.